src/Bundle/Tests/Functional/Maker/MakeGridTest.php

```diff
@@ -0,0 +1,258 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\GridBundle\Tests\Functional\Maker;
+
+use App\Entity\Book;
+use App\Entity\Price;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Bundle\MakerBundle\Exception\RuntimeCommandException;
+use Symfony\Component\Console\Tester\CommandTester;
+
+final class MakeGridTest extends MakerTestCase
+{
+    public const BOOK_GRID_PATH = 'Grid/BookGrid.php';
```

Comment:
Do we need them to be public? 
```suggestion
    private const BOOK_GRID_PATH = 'Grid/BookGrid.php';
```

Reply:
I made that change on the three constants.